wandb/sdk/service/service.py

```diff
@@ -161,13 +162,13 @@ def _launch_server(self) -> None:
                 exec_cmd_list += ["coverage", "run", "-m"]
 
             service_args = []
-            # NOTE: "wandb-core" is the name of the package that will be distributed
-            #       as the stable version of the wandb core library.
-            #
-            #       Environment variable _WANDB_CORE_PATH is a temporary development feature
-            #       to assist in running the core service from a live development directory.
-            core_path = get_core_path()
-            if core_path:
+
+            if is_require_core():
+                try:
+                    core_path = get_core_path()
+                except WandbCoreNotAvailableError as e:
+                    _sentry.reraise(e)
```

Comment:
if there's an exception, `core_path` will be `None` in the code below

Reply:
no, the re-raise will raise the exception again so we should just see the exception here and the code will fail

Reply:
or maybe i don't understand your comment here